src/Angor/Client/Pages/Create.razor

```diff
@@ -591,30 +804,29 @@
             var words = await passwordComponent.GetWalletAsync();
             var nostrKey = _derivationOperations.DeriveProjectNostrPrivateKey(words, project.ProjectIndex);
 
-            var resultId = await _RelayService.CreateNostrProfileAsync(project.Metadata.ToNostrMetadata(),
+            var resultId = await _RelayService.CreateNostrProfileAsync(
+                project.Metadata.ToNostrMetadata(),
                 NBitcoin.DataEncoders.Encoders.Hex.EncodeData(nostrKey.ToBytes()),
                 _ =>
                 {
                     createProfileSpinner = false;
 
                     if (!_.Accepted)
-                        notificationComponent.ShowErrorMessage("Failed to store the project information on the relay!!!"); //TODO add export project info
-
-                    if (nostrMetadataCreated == false)
+                    {
+                        notificationComponent.ShowErrorMessage("Failed to store the project information on the relay!");
```

Comment:
Lets start to log such errors as well? (perhaps inside the method `ShowErrorMessage`) I realise we did not do it before enough.